lib/Frontend/ParseableInterfaceSupport.cpp

```diff
@@ -257,17 +259,26 @@ static bool buildSwiftModuleFromSwiftInterface(
     std::string OutPathStr = OutPath;
     serializationOpts.OutputPath = OutPathStr.c_str();
     serializationOpts.SerializeAllSIL = true;
+    auto DTDeps = SubInstance.getDependencyTracker()->getDependencies();
+    SmallVector<std::string, 16> DepNames(DTDeps.begin(), DTDeps.end());
+    DepNames.push_back(InPath);
+    SmallVector<SerializationOptions::FileDependency, 16> Deps;
+    for (auto const &Dep : DepNames) {
+      auto DepStatus = FS.status(Dep);
+      if (!DepStatus) {
+        SubError = true;
```

Comment:
We should definitely report this error; otherwise the user won't be able to fix it. (I'm not sure how to write a test case where you have a dependency but then can't stat it, but that just means the diagnostic doesn't have to be great.)

Reply:
Ok. I'm going to look at diags next so will include this (and reporting other possible errors) in that patch.

Reply:
(done)